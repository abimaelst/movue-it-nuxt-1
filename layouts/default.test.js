import { shallowMount } from '@vue/test-utils';
import Default from './default.vue';

describe('Layouts', () => {
	it('should match default layout snapshot', () => {
		const wrapper = shallowMount(Default, {
			stubs: {
				Nuxt: true,
			},
		});

		expect(wrapper.html()).toMatchSnapshot();
	});
});
