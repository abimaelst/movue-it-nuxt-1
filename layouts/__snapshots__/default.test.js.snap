// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layouts should match default layout snapshot 1`] = `
"<div class=\\"min-h-screen max-w-5xl mx-auto flex flex-col px-5 py-8 md:py-10\\">
  <experiencebar-stub></experiencebar-stub>
  <nuxt-stub></nuxt-stub>
</div>"
`;
